docs/HowToAddnewBoard.md

```diff
@@ -0,0 +1,106 @@
+# How to add(port) a new board
+
+To support a new board, two big parts, [arch](#arch) and [configs](#configs), should be modified.
+
+## Arch
+
+This part contains BSP (Board Support Package) for chip and board where are *os/arch* folder.  
```

Comment:
This line needs to restructured. For eg.  This part contains BSP(Board Support Package) for chip and board and chip. The path to this part is *os/arch*.